src/components/UserInfo/UserInfo.scss

```diff
@@ -1 +1,3 @@
-//
```

Comment:
Use variables for colors and other repeated values. Consider creating a separate file for variables and importing it in your stylesheets.